crates/orderbook/openapi.yml

```diff
@@ -651,6 +665,31 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Auction:
+      description: |
+        A batched auction for solving.
+      type: object
+      properties:
+        block:
+          type: integer
+          description: |
+            The block number for the auction. Orders and prices are guaranteed to be valid on this
+            block. Proposed settlements should be valid for this block as well.
```

Comment:
I like this. The order set would then be those valid orders in the given block window. Seems coherent, straightforward, clear and easy to query for!